README.md

```diff
@@ -56,6 +56,21 @@ output a raw BIOS bootable disk image, or `linuxkit build -format iso-efi linuxk
 
 Since `linuxkit build` is built around the [Moby tool](https://github.com/moby/tool) the input yml files are described in the [Moby tool documentation](https://github.com/moby/tool/blob/master/docs/yaml.md).
 
+
+### Baking images
+```
+linuxkit bake linuxkit_template.yml
+```
+This will replace all angle bracketed tags in the image field by its hash defined in `build.yml`. If the tag is `<latest>` it gets replaced by the current pkg directorys git subtree hash. The location of the pkg directory is either defined by parameter `--pkgroot org:path` or in the global config, e.g.:
```

Comment:
Please document how tags other than `<latest>` are handled, or explicitly state that `<latest>` is the only supported tag today if that is the case.

Reply:
you put e.g. this into the kubernetes `build.yml` and use `<1.9>` as tag.
```
tags:
    - name: <1.9>
      hash: some_hash
```
I think this is more of a maintainer thing then for the front page.

Reply:
Whether it is for maintainers or anyone else it needs documenting, if not here then in `docs/packaging.md`.

I had not appreciated so that that this PR to have anything to do with `build.yml`. Looking back over the comments this does seem to be have been mentioned in the past and I missed it, I was working from https://github.com/linuxkit/linuxkit/pull/2648#issuecomment-355530363 onwards and it wasn't mentioned after that.

As I've mentioned before for that use case we would push tagged images to hub. i.e. rather than associating `<1.9>` with a `linuxkit/pkg:hash` in `build.yml` maintainers would just push `linuxkit/pkg:1.9` to hub.

I thought the examples of `tags` given in various comments were for inputs passed into `linuxkit build` directly (i.e. actual arguments) not to be things which it would go off and found inside a repo somewhere.

Reply:
i asked that [here](https://github.com/linuxkit/linuxkit/pull/2648#issuecomment-355347992). I still would like to keep them as they could be very interesting with #2817 and variable substitution of parameters.

Reply:
I am not convinced that there is a use case of having the tag mapping in `build.yml`, while I could see an argument for having such a mapping elsewhere as an input to `linuxkit build` (which seems to be more what #2817 is about).

In terms of the code and documentation which is in _this_ PR it needs to be self consistent and in particular the documentation completely describe the state of the functionality as it exists after this PR, not how it might look after some future PR.

Reply:
done.